torch/fft/__init__.py

```diff
@@ -93,6 +93,110 @@
     tensor([0.+0.j, 1.+0.j, 2.+0.j, 3.+0.j])
 """)
 
+fft2 = _add_docstr(_fft.fft_fft2, r"""
+fft2(input, s=None, dim=(-2, -1), norm=None) -> Tensor
+
+Computes the 2 dimensional discrete Fourier transform of :attr:`input`.
+
+Note:
+    :func:`~torch.fft.fft2` is a convenience alias for :func:`~torch.fft.fftn`
+    with new defaults for :attr:`dim`. There are no other implications (e.g. to
+    performance) of using one or the other.
+
+Note:
```

Comment:
Should there be more documentation for this function or a third sentence that simply says, "See fftn."?

Reply:
In this case the note is more specific by making it 2d-only. It also directs the user to `rfft2` instead of `rfftn`.

Reply:
OK, I can buy that.